packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -201,7 +201,7 @@ MeetingInfoUtil.getRequestBody = (options) => {
       break;
     case _PERSONAL_ROOM_:
       body.userId = destination;
-      body.orgId = ''; // TODO: when to use the org iD (fetch others PMR)
+      body.orgId = webex.credentials.getOrgId(); // TODO: when to use the org iD (fetch others PMR)
```

Comment:
What does this TODO mean?

Reply:
https://github.com/webex/webex-js-sdk/pull/2206#discussion_r744543716

@arun3528 May need your opinion on this.

Reply:
@arun3528 Need your opinion on what to do with this TODO Comment. bg info: https://github.com/webex/webex-js-sdk/pull/2206#discussion_r744543716

Reply:
I confirmed with the API team that the org Id that we need to send should correspond to that of the user for which we are trying to fetch PMR information. This code won't work when requesting PMR information of external org users, we need a different solution

Reply:
ok, yes, then this fix only works for checking the logged in user's PMR or PMRs of other users from the same org, so it's still better than nothing and fixes the most common use case for the web app, but yeah a further, better fix is required. How are you planning to proceed with this? deliver this fix and raise a new Jira for the "other org" case? or do a proper fix for all cases in this existing jira/pr?